<launch>

    <arg name="robot_nickname" value="yumi"/>
    <arg name="working_dir" default="$(find lfd_interface)/data" />

    <arg name="name" default="testdemo"/>

    <arg name="config_file" value="$(find lfd_interface)/config/demonstrations/default.yaml"/>

    <include file="$(find lfd_interface)/launch/lfd_storage.launch"/>
    
    <node name="test_node" pkg="lfd_interface" type="test_node" output="screen">
        <param name="demonstration_name" value="$(arg name)"/>
        <rosparam command="load" file="$(arg config_file)" />
        <!-- <remap from="/joint_states" to="/$(arg robot_nickname)/rws/joint_states"/> -->
    </node>

</launch>
